app/models/planning_application.rb

```diff
@@ -106,7 +106,9 @@ def timestamp_status_change
   end
 
   def days_left
-    (expiry_date - Date.current).to_i
+    days_left = Date.current.business_days_until(expiry_date)
+
+    days_left.positive? ? days_left : -expiry_date.business_days_until(Date.current)
```

Comment:
what's `-expiry_date.business_days_until(Date.current)`? Not familiar with reversing dates

Reply:
we define expiry_date as:
```
 self.expiry_date = 40.business_days.after(documents_validated_at || created_at)
 ```

so this checks how many days between the expiry date and the current date have passed, then show those days in negative because (see screenshot/ + comments for more detail) we want to inform officers of how many days past due the application is